nilearn/datasets/neurovault.py

```diff
@@ -2056,6 +2117,7 @@ def _read_download_params(
     wanted_image_ids=None, max_images=None,
     max_consecutive_fails=_MAX_CONSECUTIVE_FAILS,
     max_fails_in_collection=_MAX_FAILS_IN_COLLECTION,
+    resample=False, interpolation='continuous',
```

Comment:
Following @jdkent comment, it would be better to use `interpolation=linear` as a default.